vhdl_lang/src/syntax/subprogram.rs

```diff
@@ -442,8 +452,14 @@ end function;
         let statements = vec![code.s1("return foo + arg;").sequential_statement()];
         let body = SubprogramBody {
             specification,
-            declarations,
-            statements,
+            declarations: WithRegion {
+                item: declarations,
+                range: Position::new(0, 45).range_to(Position::new(2, 0)),
+            },
+            statements: WithRegion {
+                item: statements,
+                range: Position::new(2, 5).range_to(Position::new(4, 0)),
```

Comment:
I have really tried to avoid hard-coding positions in the test cases as it makes them very tedious to update. I have made a lot of utilities to find the position of substrings within the code that I always use.